exasol/slc_ci_setup/nox/tasks.py

```diff
@@ -1,12 +1,24 @@
 import nox
 
-from exasol.slc_ci_setup.lib.deploy_ci_build import deploy_ci_build
+from exasol.slc_ci_setup.lib.deploy_build import (
+    BuildType,
+    deploy_build,
+)
 
 
-@nox.session(name="update-github-workflows", python=False)
-def update_github_workflows(session: nox.Session):
+@nox.session(name="update-ci-github-workflows", python=False)
+def update_ci_github_workflows(session: nox.Session):
     """
     Updates the Github workflows for the SLC CI under ".github/workflows/" in this repository.
     """
-    deploy_ci_build()
+    deploy_build(build_type=BuildType.CI)
     session.run("git", "add", ".github/workflows/slc_ci*.yml")
+
+
+@nox.session(name="update-cd-github-workflows", python=False)
+def update_cd_github_workflows(session: nox.Session):
+    """
+    Updates the Github workflows for the SLC CD under ".github/workflows/" in this repository.
```

Comment:
As long as we do not have nox tasks modifying other repositories than the current one, "in this repository" is somewhat redundant:

```suggestion
    Updates the Github workflows for the SLC CD in directory ".github/workflows/".
```